locales-pending/premium.ftl

```diff
@@ -262,3 +262,80 @@ false-door-test-popup-close = Close
+# $estimated_time is the estimated time it would take for a user to complete breach resolution steps. It not be singular, and the + is meant as "or more".
+# An example of this string is Your estimated time: 15+ minutes.
+high-risk-breach-estimated-time = Your estimated time: { $estimated_time }+ minutes
+
+# Credit Card Breaches
+
+high-risk-breach-credit-card-title = Your credit card number was exposed
+high-risk-breach-credit-card-description = Anyone who gets it can make unauthorized purchases that you may be liable for. Act now to prevent financial harm.
+high-risk-breach-credit-card-step-one = If you still have this card, contact the issuer to report it stolen.
+high-risk-breach-credit-card-step-two = Request a new card with a new number.
+high-risk-breach-credit-card-step-three = Check your accounts for unauthorized charges.
+high-risk-breach-credit-card-illustration-alt = Credit card illustration
+
+# Bank Account Breaches
+
+high-risk-breach-bank-account-title = Your bank account was exposed
+high-risk-breach-bank-account-description = Taking action ASAP could give you more legal protections to help you recover any losses.
+high-risk-breach-bank-account-step-one = Notify your bank immediately that your account number has been compromised.
+high-risk-breach-bank-account-step-two = Change your account number.
+high-risk-breach-bank-account-step-three = Check your accounts for unauthorized charges.
+high-risk-breach-bank-account-illustration-alt = Bank illustration
+
+# Social Security Number Breaches
+
+high-risk-breach-social-security-title = Your social security number was exposed
+high-risk-breach-social-security-description = Scammers can open up new loans or credit cards with your social security number. Act fast to prevent financial harm.
+high-risk-breach-social-security-step-one = Protect yourself by setting up a fraud alert or freezing your credit.
+high-risk-breach-social-security-step-two = Check your credit report for unrecognized accounts.
+high-risk-breach-social-security-illustration-alt = Magnifying glass illustration
+
+# PIN Breaches 
+
+high-risk-breach-pin-title = Your PIN was exposed
+high-risk-breach-pin-description = Taking action ASAP could give you more legal protections to help you recover any losses.
+high-risk-breach-pin-step-one = Notify your bank immediately that your PIN has been compromised.
+high-risk-breach-pin-step-two = Change your PIN anywhere you’ve used the same one.
+high-risk-breach-pin-step-three = Check your accounts for unauthorized charges.
+high-risk-breach-pin-illustration-alt = PIN Illustration
+
+# No high risk breaches found
+
+high-risk-breach-none-title = Great news, we didn’t find any high risk data breaches
+# Variables
+# $num_email is number of emails that the user is monitoring for breaches.
+# $email_list is list of emails that the user is monitoring for breaches. E.g. john@yahoo.com, ali@gmail.com, sam@hotmail.com 
+high-risk-breach-none-description = { $num_email -> 
```

Comment:
This string is not a plural anymore, since you pass a list of emails.

```
high-risk-breach-none-description = We detect data breaches based on your email address, and we didn’t find any high risk data breaches for { $email_list }.
```

The comment also needs to be updated.